spec.md

```diff
@@ -295,6 +300,9 @@ The formal definition of the SMTP STS format, using [@!RFC5234], is as follows:
     sts-auri        = "rua" *WSP "=" *WSP
                        sts-uri *(*WSP "," *WSP sts-uri)
 
+    sts-id           = "id" *WSP "=" *WSP 1*16HEXDIG
```

Comment:
I think I said in email, but "ID" seems unnecessary to me since "applied policy" is a field in the report. 

That said, see Viktor's comments on the UTA mailing list about a DNS nonce, etc.